applications/nrf_desktop/src/modules/led_stream.c

```diff
@@ -23,10 +23,12 @@ LOG_MODULE_REGISTER(MODULE, CONFIG_DESKTOP_LED_STREAM_LOG_LEVEL);
 
 #define LED_ID(led) ((led) - &leds[0])
 
+#define STEPS_QUEUE_ARRAY_SIZE (CONFIG_DESKTOP_LED_STREAM_QUEUE_SIZE + 1)
```

Comment:
why +1 instead of increasing the value in Kconfig

Reply:
I wanted to make this match max free spaces in queue (current implementation could be misleading)

Reply:
Please elaborate. There seems to be a mismatch between the actual and configured queue size.

Reply:
This occurs, because queue is implemented using array and two indexes. If the indexes are equal we assume that queue is empty. We need one "extra" element to distinguish between empty and full queue.